templates/html-header.twig

```diff
@@ -30,29 +30,42 @@
 	<meta name="twitter:image" content="{{ image_data[0] }}">
 
 	{% if google_tag_value %}
-		<script>
+	<script>
+          var cookie_consent = document.cookie.split(';').filter(function(c) {return c.indexOf('greenpeace=2') >= 0}).length;
+          var google_tag_value = '{{ google_tag_value }}';
+          var enforce_cookies_policy = '{{ enforce_cookies_policy }}';
+
+          if ( enforce_cookies_policy && 1 !== parseInt( cookie_consent ) ) {
```

Comment:
@sagarsdeshmukh also here I believe enforce_cookies_policy will always be interpreted as true since it will hold a value at all times. Probably better to check if it is true.

Reply:
@kirdia ye, I checked that too, but it works because false is interpreted as '' (emtpy string) and so it doesn't enter the if statement.